docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,189 @@
+
+<br>
+<br>
+
+#### MERGE CELLS
+
+| Windows          | macOS            | Action                                          |
+| ---------------- | ---------------- | ----------------------------------------------- |
+| <kbd>Ctrl</kbd> + <kbd>M</kbd> | <kbd>Cmd</kbd> + <kbd>M</kbd> | Merge selected cells     |
+
+## Managing keyboard shortcuts
+
+It is possible to add, change and remove keyboard shortcuts by managing the registered actions programmatically using the [Shortcut Manager](@/api/shortcut-manager) API.
+
+Each keyboard shortcut action is registered in a particular context. There are three contexts:
+
+- `grid` - activated when the user is browsing the data grid (initial)
+- `editor` - activated when the user opens a cell editor
+- `menu` - activated when the user opens the cell's context menu
+
+When the user interacts presses a key or a key combination keyboard, only the actions registered for the active context are executed. Only one context is active at a time.
+
+Please keep in mind that you can get already registered contexts
+using `getContext` method and even register new context using `addContext`. To switch to another context use `setActiveContextName` method.
+
+### Removing keyboard shortcuts
+To remove an already registered keyboard shortcut (such as one of the default keyboard shortcuts), you need to search for it in the relevant context and refer to it by:
+
+- Either the key variant
+- Or the namespace
+
+Use the context's method `removeShortcutByVariants` to remove all shortcuts registered for given key variants (note that it is possible that there are multiple actions registered for a single keyboard shortcut):
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByVariants([['enter']]);
+```
+
+Use the context's method `removeShortcutByNamespace` to remove all shortcuts registered in a certain namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByNamespace('NAMESPACE_ID');
+```
+
+### Adding custom keyboard shortcuts
+
+Use the context's `addShortcut` method to register an action for a given keyboard shortcut.
+
+Within a single context, there might be multiple actions registered for the same keyboard shortcut. Your action will be simply added at the end of the stack of already defined actions.
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {} });
+```
+
+If your action must run before a certain other action, you can refer to the other action by its namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {}, position: 'before', relativeToNamespace: 'ANOTHER_NAMESPACE_ID' });
```

Comment:
@wszymanski it would be useful to show in the body of `callback: () => {}` how someone can stop the chain of execution to prevent the `ANOTHER_NAMESPACE_ID` action from executing.